extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -162,11 +157,16 @@ export class AmpAdXOriginIframeHandler {
           .then(() => this.noContent_());
     }
 
+    this.element_.appendChild(this.iframe);
+    if (opt_isA4A) {
```

Comment:
This was moved down because we want to listen for the no content message in the a4a case.